src/Integration.Vsix/SonarLintIntegrationPackage.cs

```diff
@@ -24,6 +24,12 @@ namespace SonarLint.VisualStudio.Integration.Vsix
         new string[] { "SolutionHasProjectCapability:CSharp",
                        "SolutionHasProjectCapability:VB" }
     )]
+
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", 
+        "S2931:Classes with \"IDisposable\" members should implement \"IDisposable\"", 
+        Justification = "False positive. The base class exposes a Dispose override in which the disposable instances will be disposed", 
```

Comment:
Blair and I had a discussion about this and we don't think that strictly speaking this is a false positive as _IDisposable_ is not implemented by this class or the base class. The message should be suppressed, but it would be fairer to say "By design".